### DISCUSSION TITLE
Controlling 2005 Aquacal Heater

### DISCUSSION THREAD

**Discussion by chadhobson** [tagyoureit/nodejs-poolController#782] [Nixie]
So I have an Aquacal heater from 2005. Popped the cover off it today to see what the mainboard looks like and it looks promising in a couple spots. So I hunt down some numbers off the board and I land on this PDF...

[Aquacal - Connecting External Controllers](https://www.aquacal.com/uploads/aquacal-connecting-external-controllers.pdf)

Nothing is RS-485 but there's a three terminal block at the bottom and its referenced in this PDF as being used for digital control. For the Pentair EasyTouch system on page 5 (Figure-2A), it uses two of the three terminals (Y and Z) which sounds like it might be RS-485 data pins but who knows.

Can Nixie control this heater?

**Comment by rstrouse** [comment]
Nixie can control that heater as a regular heatpump through the firemans switch and a Nixie controlled relay.  You would use interface method 1.  The 3-wire connection is actually for controllers without a heat thermostat for which Nixie has in spades.  You supply no power through this relay it is only switched by Nixie when there is a call for heat.

**Comment by chadhobson** [comment]
@rstrouse  That sounds like off/on functionality only. Is that correct? I was hoping to be able to control the digital panel on the front of it (off/on, pool/spa, temperature setting, etc) via the 2 wire control. Just not possible?

**Reply by rock-crusher** [reply]
Hi Chad,

I believe you can do everything you want through Nixie once you have everything set up including the temperature sensors,               
 link here:  [Temperature Sensors](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#temperature-sensors), Also, the heater connection here: [Gas Heaters](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#gas-heaters)

I'm sure rstrouse will chime in to help you further.

**Reply by rstrouse** [reply]
> @rstrouse That sounds like off/on functionality only. Is that correct?

Not really you will not miss RS485 control of your heater.  You will find that the pool temperature is a better guide as a thermostat and you get no additional control over the heater.  The only thing that is nice with an RS485 connection is that the error codes can be translated so you can see the text associated with the error if there is one.  Unfortunately, the text is very general anyway.  Other than that there is nothing to gain from it.

**Reply by chadhobson** [reply]
@rstrouse I'm finalizing some plumbing on my new chlorinator this weekend before doing the requisite electrical work. I'm waiting on a pressure transducer for the filter as well as well as wiring the heater's fireman's switch.

Think I can I get away with using 18 gauge solid irrigation wire to connect to my enclosure? I'm going to prewire the conduit so all I have to do is hook things up when parts arrive.

**Reply by rstrouse** [reply]
For rs485.  Sure.

**Reply by chadhobson** [reply]
Sorry for the confusion. I meant for the fireman's switch on the heater. The wiki shows it wired up to the relay hat only, so I figured its just closing a contact and not actually sending any electricity over the wire but I don't know a thing about a fireman's switch on an electric heater.

I think the irrigation wire will be fine for the rs485 connection from the chlorinator to the enclosure as well as the pressure transducer to the enclosure, its the heater I'm not so sure about.

**Reply by rock-crusher** [reply]
Hi Chad,
I would think your 18 gauge solid irrigation wire would be more than adequate for the fireman’s switch on the heater. That wire insulation is tough, made for direct burial and is UV/water resistant.
Also, stated in your heater manual is to use 18 gage LV wiring.

<img width="574" alt="Fireman's Switch" src="https://github.com/tagyoureit/nodejs-poolController/assets/94593954/72315740-7553-47d7-ba8e-4214967bb08f">

**Reply by rstrouse** [reply]
And yes it is a dry contact switch.  Do not provide any voltage to it.  When the relay engages it will close the contact and complete the circuit to the fire control module in the heater.
